### ISSUE TITLE
Should fissures/sulcus/incisure be material or immaterial?

### ISSUE THREAD

**Issue by shawntanzk** [obophenotype/uberon#2667]
Related to https://github.com/obophenotype/uberon/pull/2633

Currently it is unclear in uberon if fissures/sulcus/incisure should be material or immaterial - do they refer to the fold itself or do they refer to the space 

Currently there are examples of both. Examples: 

immaterial: 
http://purl.obolibrary.org/obo/UBERON_0014466
http://purl.obolibrary.org/obo/UBERON_0000093

material: 
terms under surface groove(http://purl.obolibrary.org/obo/UBERON_0006846): 
http://purl.obolibrary.org/obo/UBERON_0005412
http://purl.obolibrary.org/obo/UBERON_0005478

Need input on this @cmungall & @dosumis

**Comment by shawntanzk** [comment]
From UBERON call: 

In Allen Brain Atlas, it is not "represented" in the atlas - as in there is the term, but not mapped on the atlas. This should (at least in my mind) be immaterial. 
See examples by @patrick-lloyd-ray 

![Screenshot 2022-10-24 at 18 24 41](https://user-images.githubusercontent.com/76212760/197587927-6b5f6464-1429-4ee7-9623-94930222bb5d.png)
![Screenshot 2022-10-24 at 18 24 51](https://user-images.githubusercontent.com/76212760/197587956-6a003aca-9f7e-47ba-a1b4-6116eb069dc6.png)

However, @cmungall did mention that there is utility of being in the material branch as that is more used by people. 

My take: 
We follow atlas and make all the above as immaterial unless we have a use case for making it material 

Plan:
Will make a go and moving them all to immaterial and see if uberon breaks and figure out from there

**Comment by shawntanzk** [comment]
Using the following in ubergraph to find all material terms with sulcus, fissure, incisure in uberon:

<details>
<summary> SPARQL query </summary>

```SPARQL
# adding prefixes used
prefix owl: <http://www.w3.org/2002/07/owl#>
prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
prefix IAO: <http://purl.obolibrary.org/obo/IAO_>
prefix BFO: <http://purl.obolibrary.org/obo/BFO_>

# selecting only unique instances of the three variables
SELECT DISTINCT ?entity ?label WHERE
{
  # the variable property has to be definition (IAO:0000115)
  VALUES ?property {
    rdfs:label
  }
  
  # only look for uberon terms
  ?entity rdfs:isDefinedBy <http://purl.obolibrary.org/obo/uberon.owl> .
    
  # defining the order of variables in the triple
  ?entity ?property ?label .
  # entity must be material
  ?entity rdfs:subClassOf BFO:0000040
  # filtering out triples where the variable label has sulcus in it
  FILTER(contains(STR(?label), "sulcus")||contains(STR(?label), "incisure")||contains(STR(?label), "fissure"))

}
# arrange report by entity variable
ORDER BY ?entity
```
</details>

Results in the following

<details>
<summary> Table of material sulcus, incisure, fissure </summary> 

entity | label
-- | --
http://purl.obolibrary.org/obo/UBERON_0001819 | palpebral fissure
http://purl.obolibrary.org/obo/UBERON_0005412 | optic fissure
http://purl.obolibrary.org/obo/UBERON_0005478 | sulcus limitans of neural tube
http://purl.obolibrary.org/obo/UBERON_0005480 | superior orbital fissure
http://purl.obolibrary.org/obo/UBERON_0005489 | anterior interventricular sulcus
http://purl.obolibrary.org/obo/UBERON_0005490 | posterior interventricular sulcus
http://purl.obolibrary.org/obo/UBERON_0006271 | orbital fissure
http://purl.obolibrary.org/obo/UBERON_0009570 | spinal cord sulcus limitans
http://purl.obolibrary.org/obo/UBERON_0009576 | medulla oblongata sulcus limitans
http://purl.obolibrary.org/obo/UBERON_0009577 | metencephalon sulcus limitans
http://purl.obolibrary.org/obo/UBERON_0009578 | myelencephalon sulcus limitans
http://purl.obolibrary.org/obo/UBERON_0014628 | vestibular fissure of the cochlear canal
http://purl.obolibrary.org/obo/UBERON_0014870 | inferior orbital fissure
http://purl.obolibrary.org/obo/UBERON_0018339 | metotic fissure
http://purl.obolibrary.org/obo/UBERON_0036014 | gluteal sulcus
http://purl.obolibrary.org/obo/UBERON_0036212 | intertragic incisure
http://purl.obolibrary.org/obo/UBERON_2000305 | ventral sulcus
http://purl.obolibrary.org/obo/UBERON_2000589 | sulcus ypsiloniformis
http://purl.obolibrary.org/obo/UBERON_3000803 | sulcus articularis lateralis
http://purl.obolibrary.org/obo/UBERON_3000804 | sulcus articularis medialis
http://purl.obolibrary.org/obo/UBERON_3000844 | sulcus longitudinalis
http://purl.obolibrary.org/obo/UBERON_3000906 | sulcus pro musculo extensori cruris brevis
http://purl.obolibrary.org/obo/UBERON_3000908 | sulcus distalis ossis cruris
http://purl.obolibrary.org/obo/UBERON_3000909 | sulcus proximalis ossis cruris
http://purl.obolibrary.org/obo/UBERON_3010829 | sulcus intermedius


</details>

**Comment by shawntanzk** [comment]
missed groove - are they material or immaterial? I personally would like it to all be immaterial, but there's also a lot of part_of stuff with material that we need to deal with.

**Comment by shawntanzk** [comment]
<details>
<summary>Groove that are material: </summary>

entity | label
-- | --
http://purl.obolibrary.org/obo/UBERON_0004056 | primitive groove
http://purl.obolibrary.org/obo/UBERON_0005061 | neural groove
http://purl.obolibrary.org/obo/UBERON_0005455 | interventricular groove
http://purl.obolibrary.org/obo/UBERON_0006212 | bulbo-ventricular groove
http://purl.obolibrary.org/obo/UBERON_0006257 | laryngotracheal groove
http://purl.obolibrary.org/obo/UBERON_0006266 | nasolacrimal groove
http://purl.obolibrary.org/obo/UBERON_0006846 | surface groove
http://purl.obolibrary.org/obo/UBERON_0010229 | ruminant esophageal groove
http://purl.obolibrary.org/obo/UBERON_0013241 | embryonic urethral groove
http://purl.obolibrary.org/obo/UBERON_0018314 | choanal groove
http://purl.obolibrary.org/obo/UBERON_3000785 | trochlear groove of humerus
http://purl.obolibrary.org/obo/UBERON_3010175 | circumferential groove
http://purl.obolibrary.org/obo/UBERON_3010229 | costal grooves
http://purl.obolibrary.org/obo/UBERON_3010235 | nuchal groove
http://purl.obolibrary.org/obo/UBERON_3010239 | subocular groove
http://purl.obolibrary.org/obo/UBERON_3010242 | nasolabial groove
</details>

**Comment by shawntanzk** [comment]
41 terms to deal with, will make plans on this google sheets and work from there
https://docs.google.com/spreadsheets/d/1pLezAT_LHJUA5JIxszNnRdodeDwC8t7GWHUZ4xHJ0Kw/edit?usp=sharing
Probably will take awhile given they need to be manually looked and axioms to material stuff need to be changed accordingly

**Comment by shawntanzk** [comment]
just centralising stuff from https://github.com/obophenotype/uberon/pull/2689: 
1) material entity and immaterial entity are not disjoint which feels like they should
2) immaterial anatomical entity in uberon does not seem to be subclass_of immaterial entity in uberon, but in OLS it seems to be, not sure why
3) http://purl.obolibrary.org/obo/UBERON_0013118 sulcus of brain seems to be both immaterial and material (see horribly complicated graph showing route from sulcus of brain to both material and immaterial anatomical entity
![sulcus immaterial and material](https://user-images.githubusercontent.com/76212760/197787051-b044ca46-80a0-452f-8d93-5988f4e6fd56.jpg)

**Comment by cmungall** [comment]
Yes immaterial can be part of material, but not the other way around.

Debugging disjointness is one of the few times you want to omit part-ofs in graph displays - if in your diagram you ignore the blue edges it is consistent

**Comment by cmungall** [comment]
trick for finding all terms with "fissure" in label/syn and not a space:

`uberon info l~fissure .not .desc//p=i "anatomical space"`

|id|label|
|---|---|
|UBERON:0001819|palpebral fissure|
|UBERON:0005412|optic fissure|
|UBERON:0005480|superior orbital fissure|
|UBERON:0006271|orbital fissure|
|UBERON:0013445|pterygomaxillary fissure|
|UBERON:0013455|spheno-petrosal fissure|
|UBERON:0014471|primary fissure of cerebellum|
|UBERON:0014473|precentral fissure of cerebellum|
|UBERON:0014474|postcentral fissure of cerebellum|
|UBERON:0014628|vestibular fissure of the cochlear canal|
|UBERON:0014870|inferior orbital fissure|
|UBERON:0018339|metotic fissure|
|UBERON:0025128|obsolete regional part of longitudinal fissure|
